Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -328,7 +330,13 @@
 "field-token-title" = "Verification code";
 "field-username-placeholder" = "enter username";
 "field-username-title" = "Username";
-"find-in-page-number-matches" = "$1 / $2";
+"find-clear-button-accessibility-label" = "Clear find textfield";
```

Comment:
I think `textfield` should maybe be separate words. Same with `find-textfield-accessibility-label`.

Reply:
actually I noticed VoiceOver says "textfield" anyways for the component so it said it twice. I removed in a later commit :) Let me know if you notice any others, I think all the remaining textfields are in comments.

Reply:
oh good catch!